dist/macos/bundle/Barrier.app/Contents/Info.plist.in

```diff
@@ -31,6 +31,6 @@
     <key>NSHumanReadableCopyright</key>
     <string>© 2018 Debauchee Open Source Group</string>
     <key>LSMinimumSystemVersion</key>
-    <string>10.9.0</string>
+    <string>10.12.0</string>
   </dict>
```

Comment:
OK, this is fine. We should probably do a CI re-run, but I'll need to look into that. It should work fine on our CI though.

Reply:
I know that there's some desire to support older macOS versions, but our CI build agents only support up to certain older macOS releases. I'm happy to merge this change for now, but I think @p12tic wanted to support older mac releases. It may be worth opening an issue to take a look at this in more detail, but leave that with me or @p12tic :+1: